lang/kotlin/Portfile

```diff
@@ -2,8 +2,10 @@
 
 PortSystem          1.0
 PortGroup           github 1.0
+PortGroup           java 1.0
 
 github.setup        JetBrains kotlin 1.3.60 v
+revision            1
```

Comment:
Thanks for getting rid of the `kaffe` dependency. There are quite a few ports left needing similar change (or should be removed altogether), and taking care of them is a chore.

Anyone who managed to install this port likely already has Java (since `kaffe` appears to be broken) and won't notice any differences, so I wonder if a revision bump is unnecessary in this case. But I guess one reason to still bump the revision is if their Java might be too old (1.7 or lower).

Reply:
I think I'll just merge this. Bumping seems safer, or at least harmless enough.